apps/events/signals.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models.signals import pre_delete, pre_save
+from django.dispatch import receiver
+
+from .models import Attendee, EventUserAction
+
+
+@receiver(signal=pre_save, sender=Attendee)
+def handle_payment_relation_status_change(sender, instance: Attendee, **kwargs):
+    EventUserAction.objects.create(
+        user=instance.user, event=instance.event.event, type="register"
+    )
```

Comment:
This should rather use post_save and check that `created` is true https://docs.djangoproject.com/en/5.0/ref/signals/#post-save

Reply:
True that would have been better